pkg/sidecar/k8s_instance.go

```diff
@@ -178,6 +182,20 @@ func (d *K8sInstanceManager) manageContainer(ctx context.Context, container *doc
 		}
 
 		if route.Dst == nil && route.Src == nil {
+			// if default route, get the gw and add a route for DNS
```

Comment:
Any reason we're not doing the same thing we're doing with redis? That is, get the route to the DNS server (regardless of the local gateway)? (i.e., `RouteGet(...)`).

Reply:
No reason, what you suggest should work as well. I just wanted to make sure that the route is added before we remove the `default` route. That might be a candidate for refactor.